<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>publishWindow</class>
 <widget class="QDialog" name="publishWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>发起一个活动QAQ</string>
  </property>
  <widget class="QDialogButtonBox" name="publishButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>211</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="actDetail">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>350</y>
     <width>311</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_actDetail">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>活动详情</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_sports">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>运动</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_boardgame">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>桌游</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_eatout">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>聚餐</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_trip">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>出游</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_others">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>其他</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_tag">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>活动标签</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_actName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>活动名称</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="actName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="labelframe">
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>161</y>
     <width>491</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelframe_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>531</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/file/activity.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发布一个活动</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>labelframe</zorder>
  <zorder>publishButton</zorder>
  <zorder>actDetail</zorder>
  <zorder>label_actDetail</zorder>
  <zorder>checkBox_sports</zorder>
  <zorder>checkBox_boardgame</zorder>
  <zorder>checkBox_eatout</zorder>
  <zorder>checkBox_trip</zorder>
  <zorder>checkBox_others</zorder>
  <zorder>label_tag</zorder>
  <zorder>label_actName</zorder>
  <zorder>actName</zorder>
  <zorder>labelframe_2</zorder>
  <zorder>label_title</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>publishButton</sender>
   <signal>accepted()</signal>
   <receiver>publishWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>publishButton</sender>
   <signal>rejected()</signal>
   <receiver>publishWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
